tools/buildgen/copy_protos.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+# Copyright 2025 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Copy protos from proto directory to python ancillary packages directory.
+
+set -e
```

Comment:
Nit: `set -euo pipefail`. You're not currently using pipes, but pipes may be added in the future. In general, it's a good idea for all bash scripts to start with this.